catroid/src/org/catrobat/catroid/ui/ScriptActivity.java

```diff
@@ -259,9 +259,15 @@ public boolean onOptionsItemSelected(MenuItem item) {
 				break;
 
 			case R.id.unpacking:
-				Intent intent = new Intent(currentFragment.getActivity(), BackPackActivity.class);
-				intent.putExtra(BackPackActivity.EXTRA_FRAGMENT_POSITION, FRAGMENT_SOUNDS);
-				startActivity(intent);
+				if (currentFragment == lookFragment) {
```

Comment:
I'm not sure what the Intention was because the code is > 1 year old.
But i think this was on purpose because it should really be the same object (fragment).